app/src/main/java/com/company/rentafield/di/NetworkModule.kt

```diff
@@ -76,66 +78,81 @@ object NetworkModule {
         }.build()
     }
 
-    @Provides
-    @Singleton
-    fun provideService(okHttpClient: OkHttpClient): RetrofitService {
+    private inline fun <reified T> provideRetrofitService(
```

Comment:
we need to make that fun to provide Retrofit not service and inject it to other functions which provide services